app/components/post_textbox/index.js

```diff
@@ -70,10 +73,23 @@ function mapStateToProps(state, ownProps) {
         );
     }
 
+    let useGroupMentions = true;
```

Comment:
let useGroupMentions = hasLicense for ldap

Reply:
Done!